src/aat/resources/features/F-026/S-077.td.json

```diff
@@ -1,23 +1,27 @@
 {
-  "title": "must return an empty SearchResultView envelope and status code 200 if case type is not found",
+  "title": "must return an empty SearchResultView envelope and status code 200 if case type has no associated cases",
   "_guid_": "S-077",
   "_extends_": "F-026_Test_Data_Base",
 
   "specs": [
     "an active profile in CCD",
-    "uses case-reference of the case just created",
     "uses an existing case-type which doesn't have any associated cases",
     "contains an empty SearchResultView, along with an HTTP-200 OK"
   ],
+  "user": {
+    "username": "befta.caseworker.1@gmail.com",
+    "password": "Pa55word11"
+  },
   "request": {
     "pathVariables": {
-      "ctid": "AAT_AUTH_5"
+      "jid": "BEFTA_JURISDICTION_1",
+      "ctid": "DO_NOT_CREATE_CASES"
```

Comment:
DO_NOT_CREATE_CASES --> CASE_TYPE_WITH_NO_CASES
The case type name in the file should be like: "Case Type With No Cases - Don't Create"

Reply:
Updated: "ctid": "CASE_TYPE_WITH_NO_CASES"
And update name to -- Case Type With No Cases - Don't Create

Reply:
Name field allows max length of 30 chars. So shortened the name : CT With No Cases -Don’t Create